### DISCUSSION TITLE
add hdmi to finn project

### DISCUSSION THREAD

**Discussion by Ajfil** [Xilinx/finn#798] [Q&A]
hi ,i already have an finn project,it can work ,i want output video by hdmi,but no video.
here is my code
![image](https://user-images.githubusercontent.com/91005367/231415277-03ba92a3-281a-4aad-9155-85ab0dea0b3c.png)
![image](https://user-images.githubusercontent.com/91005367/231415493-25999ccb-b412-4127-8e5a-e7e204f0779f.png)

there no erro,But the code keeps running
here is phenomenon
![IMG_5716](https://user-images.githubusercontent.com/91005367/231416377-ef1adbef-e8b2-4788-a16a-2f0f952a5f2b.JPG)

the camera is working but no video
my personal idea maybe is problem of top file，when i change finn project，top file didn’t updata hdmi pins.the top_warpper.v didn't updata but top.v have hdmi pins.I don't know if that's the problem.But project can generate bitstream.
![aa76bcd6a577e16b5864f066aa89f14](https://user-images.githubusercontent.com/91005367/231423470-c9fc68d2-4989-4496-9186-f6cb38ad3ed5.png)
![e4d5187c0a78e3a6cfd181925dc0fcb](https://user-images.githubusercontent.com/91005367/231423540-4f1eb83a-6ff2-499e-bc62-bbf3b88921f9.png)
![777802424800dbef62e3db19fede408](https://user-images.githubusercontent.com/91005367/231423624-bfd96d41-d29e-414d-b80f-c24fd0cdb255.png)

or overlay of finn has other requirements .
i make changes based on https://github.com/sefaburakokcu/finn-quantized-yolo 
here is Original project's block design
![699984b30a62f30042529be911b482d](https://user-images.githubusercontent.com/91005367/231443769-b4b638f7-0d92-4561-b0ee-f3b2a396f074.png)

here is block design that i chageed
![3de66d22120ab61bb0381281ad57902](https://user-images.githubusercontent.com/91005367/231443890-2ccd99df-a93f-4999-8975-2e61a792e762.png)

i don't konw if it's problem of HP

**Comment by smtalds** [comment]
Hi  @Ajfil , How can u access camera ? When i want to access camera with `cv2.VideoCapture(0)`, its always returning `false`
